src/core/frames/morph_frame_renderer.js

```diff
@@ -0,0 +1,16 @@
+import { FrameRenderer } from "./frame_renderer"
+import { MorphElements } from "../morph_elements"
+import { dispatch } from "../../util"
+
+export class MorphFrameRenderer extends FrameRenderer {
+  static renderElement(currentElement, newParentElement) {
+    const newElement = newParentElement.children
+
+    dispatch("turbo:before-frame-morph", {
```

Comment:
What are we hoping to achieve by introducing a new event that exists separately from the other Morph events? What differs between this event and a `turbo:before-morph-element` event that references the `FrameElement` instance with its `target` and `currentElement` properties?

Reply:
@seanpdoyle That's existing behavior in main. I preserved it so as not to break anyone's usage:
https://github.com/hotwired/turbo/pull/1192/files#diff-04bb702ad80fc80a9ba0ddee175f0ad2527604ef65f45c78da9c379e5f63035eL102

Reply:
Thank you for clarifying @krschacht, I'd missed that.

@jorgemanrubia currently the `turbo:before-frame-morph` event is not yet [publicly documented](https://turbo.hotwired.dev/reference/events#page-refreshes). 

Are we willing to omit it from the public collection of events, and instead encourage `turbo:before-morph-element` events? I think it pre-dated the introduction of those events in https://github.com/hotwired/turbo/pull/1097.